src/serviceorchestration/5_0_0/sysd/serviceorchestration_sysd.tex

```diff
@@ -115,64 +115,64 @@ \subsection{Significant Prior Art}
+    \item The QualityOfService (QoS) Manager component was embedded into the Orchestrator (only QoS data comes from a support system). The current version moves these functionalities into a separate support system (which also be responsible to collect and store QoS data).
     \item X.509 certificate trust chains was used as authentication mechanism. The current version can support any type of authentication methods by using a dedicate Authentication Provider core system. 
 \end{itemize}
 
 \subsection{How This System Is Meant to Be Used}
 \label{sec:use}
 
-Service Orchestration is a recommended core system of Eclipse Arrowhead Local Cloud and is responsible for finding and pairing service consumers and providers. 
+ServiceOrchestration is a recommended core system of Eclipse Arrowhead Local Cloud and is responsible for finding and pairing service consumers and providers. 
 
 There are two ways to use the offered functionality:
 
 \begin{itemize}
-    \item An application that want to consume a service should ask the Service Orchestration to find one or more accessible providers that meet the necessary requirements. The Service Orchestration returns the information that the application needs to consume the specified service.
-    \item An application can subscribe for orchestration with the necessary requirements (or can be subscribed by an other application/support system). The Service Orchestration performs the orchestration process and returns the information on the specified channel. Additionally, whenever a system notifies the Service Orchestration system, it does the orchestration again and returns the updated information the the subscriber.
+    \item An application that want to consume a service should ask the ServiceOrchestration to find one or more accessible providers that meet the necessary requirements. The ServiceOrchestration returns the information that the application needs to consume the specified service.
+    \item An application can subscribe for orchestration with the necessary requirements (or can be subscribed by an other application/support system). The ServiceOrchestration performs the orchestration process and returns the information on the specified channel. Additionally, whenever a system notifies the ServiceOrchestration system, it does the orchestration again and returns the updated information the the subscriber.
 \end{itemize}
 
-The \textit{information} that is provided to the consumer whenever an orchestration is done depends on the strategy that the Service Orchestration system has implemented.
+The \textit{information} that is provided to the consumer whenever an orchestration is done depends on the strategy that the ServiceOrchestration system has implemented.
```

Comment:
The \textit{information} that is provided to the consumer whenever an orchestration is done depends on the strategy that the system has implemented.